python/pyspark/sql/connect/functions/builtin.py

```diff
@@ -2578,6 +2590,16 @@ def regexp_like(str: "ColumnOrName", regexp: "ColumnOrName") -> Column:
 regexp_like.__doc__ = pysparkfuncs.regexp_like.__doc__
 
 
+def randstr(length: "ColumnOrName", seed: Optional["ColumnOrName"] = None) -> Column:
```

Comment:
```suggestion
def randstr(length: "ColumnOrName", seed: Optional[Union[Column, int]] = None) -> Column:
```
It seems the `seed` should be foldable, and then cannot support column name

Reply:
Sounds good, done.